src/styles/reset.scss

```
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+}
```

to reset the settings - we need `_reset.scss` file. In order to set the default value - we better create - `_normalize.scss` file. Let's move it
```suggestion
body {
  margin: 0
}
```